### PR TITLE
add Dockerfile

### PR DESCRIPTION
**Pull request by monsur4**

### CODE REVIEW

File: Dockerfile
Comment (Lekeleebusiness) [inline Dockerfile:1]: Why did you add the Argument in the dockerfile

Comment (monsur4) [reply Dockerfile:1]: actually it is redundant. I could have just copied directly but wanted to try something else

Comment (Lekeleebusiness) [reply Dockerfile:1]: Okay no problem. Please avoid redundant approach next time.

**Review by Lekeleebusiness — APPROVED** [review]
It looks good.

**Review by ROGiwa — COMMENTED** [review]
Good.

**Review by ROGiwa — COMMENTED** [review]
g

**Review by ROGiwa — COMMENTED** [review]
g
